pages/settings/profile.tsx

```diff
@@ -18,12 +19,23 @@ export default function Settings(props) {
   const descriptionRef = useRef<HTMLTextAreaElement>();
   const avatarRef = useRef<HTMLInputElement>();
   const hideBrandingRef = useRef<HTMLInputElement>();
+  const [selectedTheme, setSelectedTheme] = useState({ value: "" });
   const [selectedTimeZone, setSelectedTimeZone] = useState({ value: props.user.timeZone });
-  const [selectedWeekStartDay, setSelectedWeekStartDay] = useState(props.user.weekStart || "Sunday");
+  const [selectedWeekStartDay, setSelectedWeekStartDay] = useState({ value: "" });
 
   const [hasErrors, setHasErrors] = useState(false);
   const [errorMessage, setErrorMessage] = useState("");
 
+  const themeOptions = [
+    {value: 'light', label: 'Light'},
+    {value: 'dark', label: 'Dark'}
+  ];
+
+  useEffect( () => {
+    setSelectedTheme(props.user.theme ? themeOptions.find( (theme) => theme.value === props.user.theme ) : null);
```

Comment:
Codacy has a fix for the issue: [Replace `props.user.theme·?·themeOptions.find(·(theme)·=>·theme.value·===·props.user.theme·)·:·null` with `⏎······props.user.theme·?·themeOptions.find((theme)·=>·theme.value·===·props.user.theme)·:·null⏎····`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
    setSelectedTheme(
```